src/cson.coffee

```diff
@@ -60,6 +60,7 @@ parseContents = (objectPath, cachePath, contents, callback) ->
   try
     object = parseObject(objectPath, contents)
   catch parseError
+    parseError.message = "#{objectPath}: #{parseError.message}"
```

Comment:
I guess what I'm trying to say is that it would be nice to follow the node conventions for errors, and include the path at the end of the message in single quotes, etc.